pkgs/tools/networking/mozillavpn/default.nix

```diff
@@ -0,0 +1,109 @@
+{ buildGoModule
+, fetchFromGitHub
+, go
+, lib
+, pkg-config
+, polkit
+, python3
+, qmake
+, qtbase
+, qtcharts
+, qtgraphicaleffects
+, qtnetworkauth
+, qtquickcontrols2
+, qttools
+, qtwebsockets
+, stdenv
+, which
+, wireguard-tools
+, wrapQtAppsHook
+}:
+
+let
+  glean_parser_4_1_1 = python3.pkgs.buildPythonPackage rec {
+    pname = "glean_parser";
+    version = "4.1.1";
+    src = python3.pkgs.fetchPypi {
+      inherit pname version;
+      hash = "sha256-4noazRqjjJNI2kTO714kSp70jZpWmqHWR2vnkgAftLE=";
+    };
+    buildInputs = with python3.pkgs; [ pytest-runner setuptools-scm ];
```

Comment:
```suggestion
    nativeBuildInputs = with python3.pkgs; [ setuptools-scm ];
```

Reply:
Upstream really requires `pytest-runner` at build time. Otherwise it tries to download it.

Reply:
Yeah but general rule is to throw pytest-runner out the window and patch it out since it is useless in nix.